docker-compose.yml

```diff
@@ -94,16 +103,17 @@ services:
       - "8080"
       - "8090"
     healthcheck:
-      test: curl -f localhost:8080/v1/brokers/list
+      test: curl -f localhost:8080/v1/brokers/list || exit -1
```

Comment:
I wonder why do you need `exit -1` here?

Reply:
Probably don't; made it consistent with the other tests but I'll check why it was like that

Reply:
Ah, actually I just tested this and remembered why now: `curl` can exit with many different error codes (ie `22` for any http 400), but docker currently expects 0 or 1. It works perfectly fine since it's only looking for 0 vs not-0, but this pattern is commonly used to constraint exit codes in case Docker ever does decide to interpret non 0 or 1 codes differently.

Reply:
OK I see, that make sense. There's no clear doc on how docker handles different error codes.
The best reference I found is https://komodor.com/learn/exit-codes-in-containers-and-kubernetes-the-complete-guide/
Does it make sense to use `exit 1` in that case?
I guess `-1/255` should work but it's not so obvious 😬 .